app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        cmd, old_file, new_file = command.split()
+    except ValueError:
```

Comment:
You can avoid nested try blocks in your case
Skip handling ValueError here and move code from `else` on the upper level. The ValueError will be caught by `except ValueError as ve: print(f"{ve}")`

Reply:
Also you can just add check if command len == 3